sources/healthdog/Cargo.toml

```
@@ -0,0 +1,32 @@
+[package]
+name = "healthdog"
+version = "0.1.0"
+authors = ["Matt Briggs <brigmatt@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2018"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+[[bin]]
+name = "healthdog"
+path = "src/main.rs"
+
+[dependencies]
+bottlerocket-release = { version = "0.1.0", path = "../bottlerocket-release"}
```

```suggestion
bottlerocket-release = { path = "../bottlerocket-release"}
```
We need to keep it in sync, and don't have plans to publish it separately, so the version shouldn't matter.

---

Fixed in [2a5aef0](https://github.com/bottlerocket-os/bottlerocket/compare/f515d383c9e181c8a531a02414b320ff9231a81f..2a5aef04d7abcae5c91bf8cec84b8e95c7ccd306)